src/tls13.c

```diff
@@ -5350,7 +5345,9 @@ int DoTls13ServerHello(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
             ssl->msgsReceived.got_hello_retry_request = 1;
             ssl->msgsReceived.got_server_hello = 0;
         }
+    }
 
+    if (args->totalExtSz > 0) {
```

Comment:
This check seems odd to me since totalExtSz is a word16 type (always 0 or greater). I guess avoids advancing args->idx by 0 in the case that totalExtSz is 0.

Reply:
The line that does the increment was previously in a different `if` block that did the same check so I added the same check to keep the old pattern